Simplenote/Classes/SPNoteEditorViewController+Extensions.swift

```diff
@@ -616,6 +616,67 @@ extension SPNoteEditorViewController {
     }
 }
 
+// MARK: - Search Map
+//
+extension SPNoteEditorViewController {
+
+    /// Show search map keyword ranges
+    ///
+    @objc
+    func showSearchMap(with searchRangeValues: [NSValue]) {
+        createSearchMapViewIfNeeded()
+        // trigger layout to get correct height
+        searchMapView.superview?.layoutSubviews()
+
+        let editorRect = noteEditorTextView.layoutManager.usedRect(for: noteEditorTextView.textContainer)
+        let editorHeight = max(editorRect.size.height,
+                               searchMapView?.frame.size.height ?? 0)
```

Comment:
`searchMapView` is used without `?` in L629 (and is guarranteed to be available). Perhaps a `guard let` below `createSearchMap..()`?

Reply:
Fixed, thanks!